### PR TITLE
Ghost URL Typo

### PR DESCRIPTION
**Pull request by daviddarnes**
Our documentation uses a specific URL pattern to ensure people are directed to the latest version of our docs, hence the removal of `v2/` in this case. Thanks!

### CODE REVIEW

**Comment by yoshinorin** [comment]
@daviddarnes 
Pending #23 

Thank you :)

**Comment by daviddarnes** [comment]
Ah I see, thanks @YoshinoriN 👍

**Comment by tomap** [comment]
#23 merged. please rebase or merge with master

**Comment by daviddarnes** [comment]
Not sure why this is still failing. I've merged with master, and added ghost.org to (what I think is) the redirects list. Anything I've missed?

File: .travis.yml [hunk 1]
```diff
@@ -4,4 +4,4 @@ rvm:
 before_script:
   - gem install awesome_bot
 script:
-  - awesome_bot README.md --allow-redirect --allow-dupe -w weibo.com,viosey.com,hackernoon.com
+  - awesome_bot README.md --allow-redirect --allow-dupe -w weibo.com,viosey.com,hackernoon.com,ghost.org
```
Comment (yoshinorin) [inline .travis.yml]: No need to add `ghost.org` domain here. These domains for `w` option.

**Review by yoshinorin — CHANGES_REQUESTED** [review]

**Comment by yoshinorin** [comment]
@daviddarnes 
CI failure caused by other reasons. I'm going to merge this PR after fix my suggestion.

File: .travis.yml (hunk 1 above)
Comment (yoshinorin) [reply .travis.yml]: So, you just do rebase or merge with master was okay :)

Comment (daviddarnes) [reply .travis.yml]: Reverted @YoshinoriN 👍

**Review by yoshinorin — APPROVED** [review]
Thank you :)

**Comment by daviddarnes** [comment]
Thank _you_ @YoshinoriN!

**Pull request merged**